tools/docker/daml-all-in-one/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM ubuntu:22.04
+
+RUN apt update
+RUN apt install curl openjdk-21-jdk -y
+RUN curl -sSL https://get.daml.com/ | sh
+ENV PATH="/root/.daml/bin:${PATH}"
+RUN daml new create-daml-app --template create-daml-app
+WORKDIR /create-daml-app
+
+RUN apt-get update && apt-get install -y supervisor
+RUN mkdir -p /var/log/supervisor
+COPY supervisord.conf /etc/supervisord.conf
+
```

Comment:
Please export the service on 0.0.0.0 instead of the default localhost via daml.yaml
Also, please generate the jwt programmatically here, export it as an env var (not a file) and use that var in the healthcheck
(Mentioning these here so that we dont accidently approve the PR until these are addressed)